apel_rest/settings.py

```diff
@@ -163,3 +163,7 @@
 
 # The introspect URL for the IAM repsonsible for token based authN/authZ
 IAM_URL = "https://iam-test.indigo-datacloud.eu/introspect"
+
+# Use these variables to revoke/grant POST rights
+ALLOWED_FOR_POST = []
+BANNED_FROM_POST = []
```

Comment:
the thinking was grammar, not variable length.

maybe 'ALLOWED_TO_POST' would be better?